css/impact.css

```diff
@@ -0,0 +1,277 @@
+*/
+.programs {
+	position: relative;
+}
+
+/* overlapping background */
+
+.programs::before {
+	position: absolute;
+	content: "";
+	z-index: -1;
+	top: 215px;
+
+	width: 100%;
+	height: 60%;
+	background: linear-gradient(
+		180deg,
+		#d3c53f 0%,
+		#d3c53f 26.04%,
+		#92a32d 58.85%,
+		#b37122 100%
+	);
+}
+
+/* layout */
+.program-gallery {
+	background-color: var(--color-onyx);
+	padding: var(--spacing-xl);
+
+	display: flex;
+	justify-content: center;
+	gap: var(--spacing-xl);
+	flex-wrap: wrap;
+
+	color: var(--text-color-light);
+}
+
+.program-gallery li {
+	width: 50%;
+	max-width: 350px;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	gap: var(--spacing-sm);
+}
+.program-gallery img {
+	max-width: 190px;
+}
+
+.program-gallery h3 {
+	font-size: var(--font-size-title-sm);
+	text-align: center;
+}
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+===================================
+*/
+
+/* custom breakpoint for programs background */
+@media (max-width: 1176px) {
+	/* 
+===================================
+Programs 
+===================================
+*/
+	/* linear gradient graphic */
+	.programs::before {
+		height: 75%;
+	}
+}
+
+/* Tablet and smaller 1024px*/
+@media (max-width: 1024px) {
+	/* 
+===================================
+Technology & Program Cards
+===================================
+*/
+
+	/* removing spacing between footer */
+	.stewardship {
+		padding-bottom: 0;
+	}
+
+	/* removing wrapper styles for mobile design */
+	.section-2-col .wrapper {
+		flex-direction: column;
+		gap: var(--spacing-lg);
+		padding: 0;
+		width: 100%;
+	}
+
+	.section-2-col .styled-img-container,
+	.section-2-col .text-container {
+		width: 100%;
+	}
+
+	/* removing absolute positioned graphic */
+	.styled-img-container::before {
+		position: static;
+	}
+
+	.styled-img-container {
+		background-color: var(--color-amaranth);
+		padding: var(--spacing-med) 0;
+	}
+
+	/* limiting img height */
+	.styled-img-container img {
+		max-height: 300px;
+	}
+
+	/* mobile wrapper */
+	.styled-img-container img,
+	.section-2-col .text-container {
+		width: var(--wrapper-width);
+		margin: 0 auto;
+	}
+}
+
+/* Tablet and smaller 768px */
+@media screen and (max-width: 768px) {
+	/* 
+===================================
+Hero 
+===================================
+*/
+	.hero {
+		/* remove background-img */
+		background: none;
+		padding: 0;
+	}
+
+	.hero-text-container {
+		width: 100%;
+		text-align: center;
+	}
+
+	/* 
+===================================
+Program Gallery
+===================================
+*/
+
+	.program-gallery {
+		padding: var(--spacing-med) var(--spacing-sm);
+	}
+
+	/* gradient graphic adjustments */
+	.programs::before {
+		top: 180px;
+	}
+}
+
+/* Mobile Only 470px */
+@media screen and (max-width: 470px) {
+	/* 
+===================================
+Mobile Fonts
+===================================
+*/
+	.technology h3,
```

Comment:
Can we replace these with `.section-2-col h3`? I think we can get rid of the content-specific classes altogether.

Reply:
ooh yes - I missed that. Thanks for pointing it out!

Reply:
I do need to close some padding on the stewardship content and the footer